### ISSUE TITLE
tweak contact requests

### ISSUE THREAD

**Issue by r10s** [deltachat/deltachat-android#1896]
with https://github.com/deltachat/deltachat-core-rust/pull/2373 merged, the behavior of handling lots of contact requests is already a bit better as they are all marked as noticed when the contact-requests-chat is opened at "menu / contact requests".

however, to make that more usable and remove some questions the user may have - while keeping the effort in ui minimal, i  suggest the following changes:

1. if `show_emails`=all is set, a tap on the contact request should not show the alert "Chat with ..." but **open the contact-requests-chat directly** (later on, we can maybe do this change also for the other `show_emails` modes)

the contact-requests-chat gets the following changes:

2. on opening, `dc_marknoticed_chat()` is _not_ called implicitly to avoid that the contact-request-item is removed from the chatlist if just "back" is pressed.

3. instead this **"Mark noticed" can be triggered explicitly by the user**, maybe by a button in the upper right corner. this is similar to "Not now" in the old alert, however, it is a "Not now for all". i do not have a good, short name for that yet, maybe just "Ok"?

4. inside each bubble or below each bubble (whatever is simpler - remember, there may be image-only messages etc.) **two buttons** are added **Block contact/Block** and **Start chat**

this issues should be targeted after https://github.com/deltachat/deltachat-android/issues/1895 and i suggest to split it into two prs, doing 4. separated. once that is done, probably also https://github.com/deltachat/deltachat-android/issues/1890 can be reopened - maybe #1890 is already fixed then as, by the dedicated buttons, we do not have this ambiguous tap-meaning in contact requests any longer.

ux wise, i think, that should improve the overall situation, esp. for emails, where easily a dozens contact requests needs to be handled. also by the dedicated buttons, there are not more taps needed as now, while having much more control - and, by resusing the already known chat view, without the needs to learn some completely new ui.

**Comment by Hocuri** [comment]
The thing is, especially for long emails it will take very long to scroll through all these emails in the contact requests.

I'd personally propose something similar to Signal's Message requests (https://signal.org/blog/message-requests/):

- When you tap on "Contact requests", another list of chats opens, just like the "Archived chats" list.
- Inside the chat, there are options to block or accept the chat.
- If that's not too difficult to do, allow the user to directly swipe a chat to the left or right to quickly dismiss it.

**Comment by r10s** [comment]
yip, that could be an additional improvement worth disussing. 

however, as you also said, nothing for the next release and nothing to keep the ui/core effort minimal :)

**Comment by r10s** [comment]
on recent irc discussions, we came to the conclusion that point 2. to 4. may be regarded as optional :)

**Comment by r10s** [comment]
k, let's regard that as done for now, if we really want additional improvements for the next release, we can file more concrete issues.
